### ISSUE TITLE
[Error] weird behavior when distrobox-host-exec need to read from stdin

### ISSUE THREAD

**Issue by 6cdh** [89luca89/distrobox#781]
**Describe the bug**

When `distrbox-host-exec` read from stdin, it don't work as expected.

I have a one-liner script `restore.sh` that automatic restore mysql data:

```shell
podman exec -i db sh -c 'exec mysql -u* -p*' < db.sql
```

But when I execute it in distrbox, it print something in `db.sql` and don't really restore data.

When I execute it in host, It don't print anything and works.

**To Reproduce**

In distrobox

```shell
$ cat < a.txt
123
$ distrobox-host-exec cat < a.txt
123
123
# hang forever
```

**Expected behavior**

I expect `distrobox-host-exec` simulate the executable whose name is given by the first command argument.

**Logs**

None

**Desktop (please complete the following information):**
 - podman 4.4.2
 - distrobox 1.4.2.1

**Comment by 89luca89** [comment]
Hi @6cdh 
does this happen if you use the `host-exec` binary directly?

**Comment by 6cdh** [comment]
When I execute `distrobox-host-exec`, it opens a shell and everything works well there.

**Comment by 89luca89** [comment]
Sorry I meant the `host-spawn` binary, not `host-exec` my bad

**Comment by 6cdh** [comment]
It's same with `distrobox-host-exec`. But it works if it runs with `-no-pty` argument, like this:
```shell
host-spawn -no-pty ...
```

**Comment by 89luca89** [comment]
Hi @6cdh 
I think this is already solved in the main branch due to f008b521ee776a66c36a87848c85d3e3f0f6a3dd

Can you try using the latest version?
For me the test you used (cat < file) works as expected

**Comment by 6cdh** [comment]
ok! it works. Thank you.
